From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: LeCraft2Ouf <robartalexis62@gmail.com>
Date: Sun, 16 Apr 2023 12:04:04 +0200
Subject: [PATCH] Test drop STONE on cactus && Remove grow cactus


diff --git a/src/main/java/net/minecraft/world/level/block/CactusBlock.java b/src/main/java/net/minecraft/world/level/block/CactusBlock.java
index 4323e5cc2054804243e8e2f24fd5447280f218de..0810ce22ff6c6da4839653d12233522e8aa075bb 100644
--- a/src/main/java/net/minecraft/world/level/block/CactusBlock.java
+++ b/src/main/java/net/minecraft/world/level/block/CactusBlock.java
@@ -2,12 +2,19 @@ package net.minecraft.world.level.block;
 
 import java.util.Iterator;
 import java.util.Random;
+
+import org.bukkit.Location;
+import org.bukkit.craftbukkit.event.CraftEventFactory; // CraftBukkit
+import org.bukkit.event.entity.CreatureSpawnEvent.SpawnReason;
+
 import net.minecraft.core.BlockPos;
 import net.minecraft.core.Direction;
 import net.minecraft.server.level.ServerLevel;
 import net.minecraft.tags.FluidTags;
 import net.minecraft.world.damagesource.DamageSource;
 import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.item.ItemEntity;
+import net.minecraft.world.item.ItemStack;
 import net.minecraft.world.level.BlockGetter;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.LevelAccessor;
@@ -21,7 +28,6 @@ import net.minecraft.world.level.material.Material;
 import net.minecraft.world.level.pathfinder.PathComputationType;
 import net.minecraft.world.phys.shapes.CollisionContext;
 import net.minecraft.world.phys.shapes.VoxelShape;
-import org.bukkit.craftbukkit.event.CraftEventFactory; // CraftBukkit
 
 public class CactusBlock extends Block {
 
@@ -54,19 +60,24 @@ public class CactusBlock extends Block {
             for (i = 1; world.getBlockState(pos.below(i)).is((Block) this); ++i) {
                 ;
             }
-
-            if (i < world.paperConfig.cactusMaxHeight) { // Paper - Configurable growth height
-                int j = (Integer) state.getValue(CactusBlock.AGE);
-
-                if (j >= (byte) range(3, ((100.0F / world.spigotConfig.cactusModifier) * 15) + 0.5F, 15)) { // Spigot
-                    CraftEventFactory.handleBlockGrowEvent(world, blockposition1, this.defaultBlockState()); // CraftBukkit
-                    BlockState iblockdata1 = (BlockState) state.setValue(CactusBlock.AGE, 0);
-
-                    world.setBlock(pos, iblockdata1, 4);
-                    iblockdata1.neighborChanged(world, blockposition1, this, pos, false);
-                } else {
-                    world.setBlock(pos, (BlockState) state.setValue(CactusBlock.AGE, j + 1), 4);
-                }
+            Location location = new Location(world.getWorld(), pos.getX(), pos.getY(), pos.getZ());
+            ItemStack item = new ItemStack(Blocks.STONE, 1);
+            ItemEntity entity = new ItemEntity(world, location.getX(), location.getY(), location.getZ(), item);
+            world.addEntity(entity, SpawnReason.CUSTOM);
+            
+
+
+
+            //if (i < world.paperConfig.cactusMaxHeight) { // Paper - Configurable growth height
+            //    int j = (Integer) state.getValue(CactusBlock.AGE);
+            //    if (j >= (byte) range(3, ((100.0F / world.spigotConfig.cactusModifier) * 15) + 0.5F, 15)) { // Spigot
+            //        CraftEventFactory.handleBlockGrowEvent(world, blockposition1, this.defaultBlockState()); // CraftBukkit
+            //        BlockState iblockdata1 = (BlockState) state.setValue(CactusBlock.AGE, 0);
+            //        world.setBlock(pos, iblockdata1, 4);
+            //        iblockdata1.neighborChanged(world, blockposition1, this, pos, false);
+            //    } else {
+            //        world.setBlock(pos, (BlockState) state.setValue(CactusBlock.AGE, j + 1), 4);
+            //    }
 
             }
         }
