From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: LeCraft2Ouf <robartalexis62@gmail.com>
Date: Sat, 2 Dec 2023 03:28:08 +0100
Subject: [PATCH] Remove drop cactus & Remove RandomTick Cactus


diff --git a/src/main/java/net/minecraft/world/level/block/CactusBlock.java b/src/main/java/net/minecraft/world/level/block/CactusBlock.java
index 4ac3e80737546994603f46bb37c811a588566394..1ce855b9e596a646cc67f78d7ff5e4d6aa6f0f3f 100644
--- a/src/main/java/net/minecraft/world/level/block/CactusBlock.java
+++ b/src/main/java/net/minecraft/world/level/block/CactusBlock.java
@@ -49,37 +49,6 @@ public class CactusBlock extends Block {
 
     }
 
-    @Override
-    public void randomTick(BlockState state, ServerLevel world, BlockPos pos, Random random) {
-        BlockPos blockposition1 = pos.above();
-
-        if (world.isEmptyBlock(blockposition1)) {
-            int i;
-
-            for (i = 1; world.getBlockState(pos.below(i)).is((Block) this); ++i) {
-                ;
-            }
-            if (i < world.paperConfig.cactusMaxHeight) { // Paper - Configurable growth height
-                ItemStack item = new ItemStack(Blocks.CACTUS, 1);
-                ItemEntity entity = new ItemEntity(world, blockposition1.getX(), blockposition1.getY(), blockposition1.getZ(), item);
-                entity.pickupDelay = 10;
-                world.getWorld().addEntity(entity, SpawnReason.CUSTOM);
-                
-
-            //    int j = (Integer) state.getValue(CactusBlock.AGE);
-            //    if (j >= (byte) range(3, ((100.0F / world.spigotConfig.cactusModifier) * 15) + 0.5F, 15)) { // Spigot
-            //        CraftEventFactory.handleBlockGrowEvent(world, blockposition1, this.defaultBlockState()); // CraftBukkit
-            //        BlockState iblockdata1 = (BlockState) state.setValue(CactusBlock.AGE, 0);
-            //        world.setBlock(pos, iblockdata1, 4);
-            //        iblockdata1.neighborChanged(world, blockposition1, this, pos, false);
-            //    } else {
-            //        world.setBlock(pos, (BlockState) state.setValue(CactusBlock.AGE, j + 1), 4);
-            //    }
-
-            }
-        }
-    }
-
     @Override
     public VoxelShape getCollisionShape(BlockState state, BlockGetter world, BlockPos pos, CollisionContext context) {
         return CactusBlock.COLLISION_SHAPE;
