From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: LeCraft2Ouf <leboss6227@gmail.com>
Date: Sun, 24 Mar 2024 18:43:59 +0100
Subject: [PATCH] Ajout terracotta block can spawn pumpking


diff --git a/src/main/java/net/minecraft/world/level/block/StemBlock.java b/src/main/java/net/minecraft/world/level/block/StemBlock.java
index 4a45dccc06e59eb1cf84732a8099f920405012e5..6a99bda518b288e406910f2f79751e055ef91586 100644
--- a/src/main/java/net/minecraft/world/level/block/StemBlock.java
+++ b/src/main/java/net/minecraft/world/level/block/StemBlock.java
@@ -64,7 +64,7 @@ public class StemBlock extends BushBlock implements BonemealableBlock {
                     BlockState iblockdata1 = world.getBlockState(blockposition1.below());
 
                     if (this == Blocks.PUMPKIN_STEM){
-                        if (world.getBlockState(blockposition1).isAir() && (iblockdata1.is(Blocks.FARMLAND) || iblockdata1.is(BlockTags.DIRT) || iblockdata1.is(Blocks.WHITE_GLAZED_TERRACOTTA))) {
+                        if (world.getBlockState(blockposition1).isAir() && (iblockdata1.is(Blocks.FARMLAND) || iblockdata1.is(BlockTags.DIRT) || iblockdata1.is(Blocks.WHITE_GLAZED_TERRACOTTA) || iblockdata1.is(Blocks.ORANGE_GLAZED_TERRACOTTA) || iblockdata1.is(Blocks.MAGENTA_GLAZED_TERRACOTTA) || iblockdata1.is(Blocks.LIGHT_BLUE_GLAZED_TERRACOTTA))) {
                         // CraftBukkit start
                             if (!CraftEventFactory.handleBlockGrowEvent(world, blockposition1, this.fruit.defaultBlockState())) {
                                 return;
