From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: LeCraft2Ouf <leboss6227@gmail.com>
Date: Thu, 11 Apr 2024 23:35:56 +0200
Subject: [PATCH] Remove cactus on cactus block


diff --git a/src/main/java/net/minecraft/world/level/block/CactusBlock.java b/src/main/java/net/minecraft/world/level/block/CactusBlock.java
index 7182820928d25bec910fe63034d8a4fba4623f10..6d6f393cb360d5e2020f6199a24e7290134a6726 100644
--- a/src/main/java/net/minecraft/world/level/block/CactusBlock.java
+++ b/src/main/java/net/minecraft/world/level/block/CactusBlock.java
@@ -70,7 +70,7 @@ public class CactusBlock extends Block {
          do {
              if (!iterator.hasNext()) {
                  BlockState iblockdata1 = world.getBlockState(pos.below());
-                 return (iblockdata1.is(Blocks.CACTUS) || iblockdata1.is(Blocks.SAND) || iblockdata1.is(Blocks.RED_SAND)) && !world.getBlockState(pos.above()).getMaterial().isLiquid();
+                 return iblockdata1.is(Blocks.SAND) || iblockdata1.is(Blocks.RED_SAND)) && !world.getBlockState(pos.above()).getMaterial().isLiquid();
              }
 
              enumdirection = (Direction) iterator.next();
