From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: LeCraft <robartalexis62@gmail.com>
Date: Mon, 2 Sep 2024 10:21:08 +0200
Subject: [PATCH] Try cancel grow Melon Block


diff --git a/src/main/java/net/minecraft/world/level/block/StemBlock.java b/src/main/java/net/minecraft/world/level/block/StemBlock.java
index be0e9ca885e8bc62cc66ca781b0282e020c83d9e..05899325011f9299baeee0424723c5e7e16d8bf6 100644
--- a/src/main/java/net/minecraft/world/level/block/StemBlock.java
+++ b/src/main/java/net/minecraft/world/level/block/StemBlock.java
@@ -71,6 +71,8 @@ public class StemBlock extends BushBlock implements BonemealableBlock {
                             }
                         }
                         // CraftBukkit end
+                    } else if (this == Blocks.MELON){
+                        return;    
                     } else {
                         if (world.getBlockState(blockposition1).isAir() && (iblockdata1.is(Blocks.FARMLAND) || iblockdata1.is(BlockTags.DIRT))) {
                         // CraftBukkit start
